docs/reference/modules/cluster/misc.asciidoc

```diff
@@ -127,7 +126,7 @@ PUT /_cluster/settings
 
 
 [[persistent-tasks-allocation]]
-==== Persistent Tasks Allocations
+===== Persistent tasks allocations
 
```

Comment:
The plural here seems odd--and "a kind of tasks" seems just wrong. (As does "long-live tasks".)

Reply:
Thanks. Updated both.